### ISSUE TITLE
[Problem] Cant Connect to Sonarr / Radarr Service due to CORS (even after bug 250 Fix)

### ISSUE THREAD

**Issue by mokshmridul** [bastienwirtz/homer#272]
Im trying to connect to the custom Sonarr and Radarr Service, but im getting the following errors in the console:

```
homer.example.in/:1 Access to fetch at 'https://sonarr.example.in/api/queue' from origin 'https://homer.example.in' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'.
```

My Homer Config for Sonarr is below:

```
      - name: "Sonarr"
        logo: "assets/sonarr.png"
        type: "Sonarr"
        apikey: "b2c42612345678fead3ebbad3f8cfa12"
        subtitle: "Automatically download TV Shows"
        tag: ""
        url: "https://sonarr.example.in"
        target: '_blank'
```

My NGINX Sonarr Block

```
#Sonarr Block
#Add entry in Cloudflare DNS ("CNAME sonarr example.in") to enable
server {
  #BlockedAgent
  if ($blockedagent) {
        return 403;
  }
  
  #Bad Bots Filtering
  if ($limit_bots = 1) {
            return 403;
  }
  
  #listen [::]:443 ssl http2;
  listen 443 ssl http2;
  server_name sonarr.example.in;
  include /config/nginx/conf.d/*.conf;
  include /config/nginx/cloudflare;
  add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;

    #SSL Configuration
    include /config/nginx/ssl.conf;

    location / {
      proxy_pass http://192.168.0.11:8989;
      proxy_set_header Host $host;
      proxy_set_header X-Real-IP $remote_addr;
      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    }
}
```

**Comment by rvankraaij** [comment]
The fix for bug #250 was merged but AFAIK not released yet, so we will have to wait until that's done.

**Comment by mokshmridul** [comment]
> The fix for bug #250 was merged but AFAIK not released yet, so we will have to wait until that's done.

Ahh okay, my bad. I thought this got released. Im going to mark this thread as closed for now, in this case.

Thanks.

**Comment by chivagio** [comment]
Apologies for reviving this topic. Is it possible the fix was not released yet ? I have the exact same issue as described above.
